bazel/repository_locations.bzl

```diff
@@ -898,7 +898,7 @@ REPOSITORY_LOCATIONS_SPEC = dict(
             "envoy.filters.network.wasm",
             "envoy.stat_sinks.wasm",
         ],
-        last_updated = "2020-07-29",
+        last_updated = "2020-10-09",
```

Comment:
`last_updated = "2018-12-18",`

Reply:
I reckon maybe skip fixing in this PR, I have a followup for `release_date` that I will post shortly.

Reply:
Sounds good but this is a good example of why `release_date` makes sense. The dep in question was released at the end of 2018 so having reference to `last_updated` as 2020 could misleed people into thinking the release is current (it isn't as 4.8 has been released however apparently there are conflicts with Envoy WASM ATM).